website/src/pages/jobs.tsx

```diff
@@ -3,9 +3,55 @@ import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
 import ArrowLeftIcon from 'mdi-react/ArrowLeftIcon'
 import Layout from '../components/Layout'
 import Carousel from 'react-bootstrap/Carousel'
+import { QuoteCarousel } from '../components/QuoteCarousel'
 import { ContentSection } from '../components/content/ContentSection'
 
 const CareersPage = props => {
+    const glassdoorReviewItems = [
+        {
+            header: 'Just interviewed and accepted role / position',
+            quote: 'Transparency and openness are genuine and real differentiators. Highly collaborative and focused exec team.',
+            by: 'Current Employee, Nov 23, 2021 - ',
+            linkText: 'Read full review',
+            link: 'https://www.glassdoor.com/Reviews/Employee-Review-Sourcegraph-RVW55781366.htm',
+        },
+        {
+            header: 'Progressive company that means what they say',
+            quote: 'Progressive company that means what they say. For the first time ever.. i have no complaints about my place of employment',
+            by: 'Current Employee, Sep 3, 2021 - ',
+            linkText: 'Read full review',
+            link: 'https://www.glassdoor.com/Reviews/Employee-Review-Sourcegraph-RVW52059760.htm',
+        },
+        {
+            header: 'Fast paced startup that cares',
+            quote: 'Great teams full of extremely smart people. Management that cares about the well being of its employees. Full remote and async culture.',
+            by: 'Current Employee, Feb 4, 2021 - ',
+            linkText: 'Read full review',
+            link: 'https://www.glassdoor.com/Reviews/Employee-Review-Sourcegraph-RVW41989078.htm',
+        },
+        {
+            header: 'Great start to an awesome adventure',
+            quote: '- Professional at a level many pre-IPO companies are not - Really good product that is easy to get behind and support - Dedicated leadership that support all employees - Everyone is motivated and pulling in the same direction',
```

Comment:
Can you remove the dash and space `- ` at the beginning of this quote?

Reply:
Ah, yea I meant to adjust the bullet point quote. Done ✅